anchor-contributor/tests/helpers/contributor.ts

```diff
@@ -212,19 +231,8 @@ export class IccoContributor {
 
     const custodyOrWrappedMeta = await (async () => {
       const mintInfo = await getMint(program.provider.connection, acceptedMint);
-      if (mintInfo.mintAuthority == tokenMintSigner) {
-        //First derive the Wrapped Mint Key
-        const nativeInfo = await getOriginalAssetSol(
-          program.provider.connection,
-          tokenBridge.toString(),
-          acceptedMint.toString()
-        );
-        const wrappedMintKey = deriveAddress(
-          [Buffer.from("wrapped"), serializeUint16(nativeInfo.chainId as number), acceptedMint.toBytes()],
-          tokenBridge
```

Comment:
1. This was wrong (should have been using the token address from `nativeInfo`
2. `wrappedMintKey` will always equal `acceptedMint` anyway